src/Castle.Core.Tests/Castle.Core.Tests.csproj

```diff
@@ -20,7 +20,7 @@
 		<StartupObject>Program</StartupObject>
 	</PropertyGroup>
 
-	<ItemGroup Condition="'$(TargetFramework)'=='netcoreapp1.1'">
+	<ItemGroup Condition="'$(TargetFramework)'=='netcoreapp2.0' or '$(TargetFramework)'=='netcoreapp3.1'">
```

Comment:
`netcoreapp2.0` &rarr; `netcoreapp2.1`, since you already changed the version elsewhere.

(Since the build works even with that typo, I'm wondering if we actually still need all these dependency for `netcoreapp2.x` and `netcoreapp3.x`. Would be nice to get rid of as many of them as we can.)